ghost/members-api/lib/repositories/MemberRepository.js

```diff
@@ -211,6 +212,16 @@ module.exports = class MemberRepository {
         }, options);
     }
 
+    async _generateTransientId() {
```

Comment:
Why is this async?

Reply:
I had async token generators in mind, but currently it is just implemented with a sync uuid. So my thinking was that generators are often async, so the method was also marked like that. But since it is a private method here, it doesn't really matter and I was probably overthinking 😅